### ISSUE TITLE
Responsive Design

### ISSUE THREAD

**Issue by CodyMichaelSimmons** [CodyMichaelSimmons/EmployeeWellnessProject#29]
Now that the app has a very basic interface, a responsive design can be created. The site should be accessible on mobile browsers and on a laptop. Accessibility across all browsers can be worried about later once it's working sufficiently on the major ones.
